java/ql/lib/ext/java.nio.file.model.yml

```diff
@@ -34,3 +34,22 @@ extensions:
       - ["java.nio.file", "Path", True, "toUri", "", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
       - ["java.nio.file", "Paths", True, "get", "", "", "Argument[0]", "ReturnValue", "taint", "manual"]
       - ["java.nio.file", "Paths", True, "get", "", "", "Argument[1].ArrayElement", "ReturnValue", "taint", "manual"]
+  - addsTo:
+      pack: codeql/java-all
+      extensible: neutralModel
+    data:
+      - ["java.nio.file", "Files" "exists", "", "manual"]
+      - ["java.nio.file", "Files" "getLastModifiedTime", "", "manual"]
+      - ["java.nio.file", "Files" "getOwner", "", "manual"]
+      - ["java.nio.file", "Files" "getPosixFilePermissions", "", "manual"]
+      - ["java.nio.file", "Files" "isDirectory", "", "manual"]
+      - ["java.nio.file", "Files" "isExecutable", "", "manual"]
+      - ["java.nio.file", "Files" "isHidden", "", "manual"]
+      - ["java.nio.file", "Files" "isReadable", "", "manual"]
+      - ["java.nio.file", "Files" "isRegularFile", "", "manual"]
+      - ["java.nio.file", "Files" "isSameFile", "", "manual"]
+      - ["java.nio.file", "Files" "isSymbolicLink", "", "manual"]
+      - ["java.nio.file", "Files" "isWritable", "", "manual"]
+      - ["java.nio.file", "Files" "notExists", "", "manual"]
+      - ["java.nio.file", "Files" "setLastModifiedTime", "", "manual"]
+      - ["java.nio.file", "Files" "size", "", "manual"]
```

Comment:
```suggestion
      - ["java.nio.file", "Files", "exists", "", "manual"]
      - ["java.nio.file", "Files", "getLastModifiedTime", "", "manual"]
      - ["java.nio.file", "Files", "getOwner", "", "manual"]
      - ["java.nio.file", "Files", "getPosixFilePermissions", "", "manual"]
      - ["java.nio.file", "Files", "isDirectory", "", "manual"]
      - ["java.nio.file", "Files", "isExecutable", "", "manual"]
      - ["java.nio.file", "Files", "isHidden", "", "manual"]
      - ["java.nio.file", "Files", "isReadable", "", "manual"]
      - ["java.nio.file", "Files", "isRegularFile", "", "manual"]
      - ["java.nio.file", "Files", "isSameFile", "", "manual"]
      - ["java.nio.file", "Files", "isSymbolicLink", "", "manual"]
      - ["java.nio.file", "Files", "isWritable", "", "manual"]
      - ["java.nio.file", "Files", "notExists", "", "manual"]
      - ["java.nio.file", "Files", "setLastModifiedTime", "", "manual"]
      - ["java.nio.file", "Files", "size", "", "manual"]
```

Reply:
Thanks for catching these typos! Updated in https://github.com/github/codeql/pull/12249/commits/0b99aa0bd3538944a517b82ca38bd7ee72e95bd5.